qa/suites/rados/singleton-nomsgr/all/ceph-kvstore-tool.yaml

```diff
@@ -0,0 +1,12 @@
+overrides:
+  ceph:
+    log-whitelist:
+    - but it is still running
+    - overall HEALTH_
+    - \(POOL_APP_NOT_ENABLED\)
+
+tasks:
+- workunit:
```

Comment:
need "- install:" task before this to install the packages.  shouldnt' need ceph task, tho, since you don't need a running cluster or ceph.conf.  which means the log-whitelist lines above can go away.

Reply:
thank you, I will add a "- isntall:" task. 

it seems that our tools need `ceph.conf`( ceph-kvstore-tool searches a ceph.conf at first)

Reply:
updated, Sage. is it right now?

Reply:
let's find out!